deb/after_remove.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+rm /usr/local/bin/miner
```

Comment:
nit: maybe do `rm -f` here just to be sure it isn't interactive? Probably not needed.

Reply:
good catch. updated.